openedx/core/djangoapps/content/search/documents.py

```diff
@@ -164,10 +167,11 @@ def _tags_for_content_object(object_id: UsageKey | LearningContextKey) -> dict:
 
     See the comments above on "Field.tags" for an explanation of the format.
 
-    e.g. for something tagged "Difficulty: Hard" and "Location: Vancouver" this
-    would return:
+    e.g. for something tagged "Difficulty: Hard" and "Location: Vancouver",
+    and in Collections 3 and 4, this would return:
         {
             "tags": {
+                "collections": [3, 4],
```

Comment:
I'm still not thrilled with storing collections together with tags. If we store it in the same property, I think we couldn't use the `reason` field to do partial updates inside this doc property (although we can do partial updates at the doc level).

Despite being synchronous, I will not object because these operations are cheap. I'm just failing to see any advantage in this approach. :thinking:

Reply:
I would make it it's own field, unless that's a difficult change?

Reply:
Will do, it's not difficult at all.